docs-ref-services/latest/communication-phone-numbers-readme.md

```diff
@@ -49,21 +54,27 @@ All geographic phone numbers within the same country are grouped into a phone pl
 
 ### Searching and acquiring numbers
 
-Phone numbers can be searched through the search creation API by providing a phone number type (geographic or toll-free), assignment type (person or application), calling and sms capabilities, an area code and quantity of phone numbers. The provided quantity of phone numbers will be reserved for 15 minutes. This search of phone numbers can either be cancelled or purchased. If the search is cancelled, then the phone numbers will become available to others. If the search is purchased, then the phone numbers are acquired for the Azure resource.
+Phone numbers can be searched through the search creation API by providing a phone number type (geographic or toll-free), assignment type (person or application), calling and sms capabilities, an area code and quantity of phone numbers. The provided quantity of phone numbers will be reserved for 15 minutes. This search of phone numbers can either be canceled or purchased. If the search is canceled, then the phone numbers become available to others. If the search is purchased, then the phone numbers are acquired for the Azure resource.
 
 ### Configuring phone numbers
 
 Phone numbers can have a combination of capabilities. They can be configured to support inbound and/or outbound calling, or neither if you won't use the phone number for calling. The same applies to sms capabilities.
 
 It is important to consider the assignment type of your phone number. Some capabilities are restricted to a particular assignment type.
 
+## Sip routing client
+
+Direct routing feature allows connecting customer-provided telephony infrastructure to Azure Communication Resources. In order to set up routing configuration properly, customer needs to supply the SIP trunk configuration and SIP routing rules for calls. SIP routing client provides the necessary interface for setting this configuration.
+
+When the call arrives, system tries to match the destination number with regex number patterns of defined routes. The first route to match the number will be selected. The order of regex matching is the same as the order of routes in configuration, therefore the order of routes matters. Once a route is matched, the call is routed to the first trunk in the route's trunks list. If the trunk is not available, next trunk in the list is selected.
```

Comment:
Should it be "When you make a call"? instead of "When the call arrives", which sounds more like an inbound call?

Reply:
After agreement will put "When the call is made" instead of "When you make a call"